src/main/java/bio/terra/cli/utils/Printer.java

```diff
@@ -20,16 +21,19 @@
 
   private static final PrintStream DEFAULT_OUT_STREAM = System.out;
   private static final PrintStream DEFAULT_ERR_STREAM = System.err;
+  private static final InputStream DEFAULT_IN_STREAM = System.in;
 
   private final PrintStream out;
   private final PrintStream err;
+  private final InputStream in;
 
   private static Printer printer;
 
   /** Constructor that initializes the printer with the specified output streams. */
-  private Printer(PrintStream out, PrintStream err) {
+  private Printer(PrintStream out, PrintStream err, InputStream in) {
```

Comment:
Maybe need to rename the class now that it's reading in addition to printing.

Reply:
Renamed to `UserIO`.